Packs/Campaign/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Phishing Campaign",
     "description": "This pack can help you find related phishing, spam or other types of email incidents and characterize campaigns.",
     "support": "xsoar",
-    "currentVersion": "3.1.5",
+    "currentVersion": "3.2.0",
```

Comment:
If this is just fixing a bug should be a revision no?

Reply:
It's a big bugfix (which also adds a new dependency) + layout overhaul, but I changed to revision, maybe we can discuss it now on our call